app/utils/CheckYourAnswersHelper.scala

```diff
@@ -499,38 +499,38 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
           userAnswers.get(ClaimantTypePage).contains(ClaimantType.Representative) match {
           case true =>
             (userAnswers.get(ClaimantTypePage).contains(ClaimantType.Representative) match {
-              case true => (userAnswers.get(WhomToPayPage) match {
+              case true => userAnswers.get(WhomToPayPage) match {
                 case None => Seq.empty
-                case _ => Seq(whomToPay.get)
-              })
+                case _ => getAnswerRow(whomToPay)
+              }
               case _ => Seq.empty
             }) ++
               (userAnswers.get(ClaimantTypePage).contains(ClaimantType.Representative) match {
-                case true => (userAnswers.get(WhomToPayPage).contains(WhomToPay.Representative) match {
-                  case true => Seq(indirectRepresentative.get)
+                case true => userAnswers.get(WhomToPayPage).contains(WhomToPay.Representative) match {
+                  case true => getAnswerRow(indirectRepresentative)
                   case _ => Seq.empty
```

Comment:
Use Nil instead of Seq.empty . _It’s more “idiomatic Scala” Scala to use Nil rather than List()_

Reply:
fixed it